jetty-client/src/main/java/org/eclipse/jetty/client/http/HttpSenderOverHTTP.java

```diff
@@ -233,6 +235,11 @@ protected Action process() throws Exception
                             chunkBuffer == null ? -1 : chunkBuffer.remaining(),
                             contentBuffer == null ? -1 : contentBuffer.remaining(),
                             result, generator);
+                if (BufferUtil.hasContent(headerBuffer))
+                    httpChannelOverHTTP.getHttpConnection().addBytesOut(headerBuffer.remaining() );
+                if (BufferUtil.hasContent(contentBuffer))
```

Comment:
I think it would be more efficient to do (assuming `httpConnection` as field and `LongAdder` used ):
```java
httpConnection.add(BufferUtil.length(headerBuffer)+BufferUtil.length(chunkBuffer)+BufferUtil.length(contentBuffer));
```
This only updates the LongAdder once and only access the pos/limit of each buffer once, with suitable null checks.

It also includes the chunked bytes, which I think it should do.